Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -74,6 +76,51 @@ void JitArm64::rfi(UGeckoInstruction inst)
   gpr.Unlock(WA);
 }
 
+void JitArm64::WriteBranchWatch(u32 origin, u32 destination, UGeckoInstruction inst,
+                                BitSet32 gprs_in_use, BitSet32 fprs_in_use)
+{
+  // Use ARM64Reg::X0 as a temporary to avoid moving it later in ABI_CallFunction.
+  static constexpr ARM64Reg branch_watch = ARM64Reg::X0;
+  gpr.Lock(EncodeRegTo32(branch_watch));
+
+  MOVP2R(branch_watch, &m_branch_watch);
+  const ARM64Reg WA = gpr.GetReg();
+  LDRB(IndexType::Unsigned, WA, branch_watch,
+       static_cast<int>(Core::BranchWatch::GetOffsetOfRecordingActive()));
+  FixupBranch branch = TBNZ(WA, true);
```

Comment:
Using `CBNZ` instead of `TBNZ` would be a bit more idiomatic. I don't think it actually affects performance, though.

Reply:
CBZ, actually. I'm not sure how it was working with TBNZ, and my testing capabilities are limited on Android builds to find out.